hummingbot/templates/conf_dev_4_twap_strategy_TEMPLATE.yml

```diff
@@ -0,0 +1,35 @@
+########################################################
+###       Execution4 strategy config               ###
+########################################################
+
+# The following configuations are only required for the
+# twap strategy
```

Comment:
We've added a required field "template_version" to config templates recently. Please add that in. You can refer to other template files in the templates directory for how to do it.

Reply:
ok